src/Components/Components/src/RenderTree/RenderTreeEdit.cs

```diff
@@ -53,12 +53,14 @@ namespace Microsoft.AspNetCore.Components.RenderTree
         private RenderTreeEdit(RenderTreeEditType type) : this()
         {
             Type = type;
+            RemovedAttributeName = default!;
```

Comment:
Should we change the declaration of this field to be nullable instead?

It not only can be null, it usually is null.

Reply:
I tried this at first but ran into some annoying issues with the public APIs analyzer. After a night's sleep, turns out we have to make some updates to the PublicShipped.API.txt files under both the Component and Ignitor project since we compile include source files from the components project in the ignitor project.